index.html

```diff
@@ -303,4 +303,4 @@ <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
 <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
- [ ] Scripts are best put in the <head> as this  allows it to load before your page. I suggest you implement that as this makes your code more semantic

Reply:
##OPTIONAL
- [ ] Nice Job! Remember to use words like "Please, Kindly". Please refer to my previous comment above ✅ 
- [ ] Kindly use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.